src/Dirstructure.cpp

```diff
@@ -214,7 +217,7 @@ wxString Dirstructure::MaximaDefaultLocation()
     return maximaLocation;
 
   wxLogMessage(wxString::Format(notFound,maximaLocation.utf8_str()));
-  maximaLocation = "/opt/local/bin/maxima";
+  maximaLocation = OSX_MACPORTS_PREFIX "/bin/maxima";
```

Comment:
Perhaps we should rename this to MAXIMA_PREFIX, as fink and homebrew probably need a similar setting... ...and we could document this setting in README.md